React/CoreModules/RCTDevSettings.mm

```diff
@@ -186,16 +191,34 @@ - (void)initialize
         }
                          queue:dispatch_get_main_queue()
                      forMethod:@"reload"];
+    #if RCT_DEV_MENU
+       _bridgeExecutorOverrideTokenDevMenu = [[RCTPackagerConnection sharedPackagerConnection]
+           addNotificationHandler:^(id params) {
+             if (params != (id)kCFNull && [params[@"debug"] boolValue]) {
+               weakBridge.executorClass = objc_lookUpClass("RCTWebSocketExecutor");
+             }
+           }
+                            queue:dispatch_get_main_queue()
+                        forMethod:@"devMenu"];
```

Comment:
Do we actually need this? Your method doesn't have any params, and doesn't need to change the executor used.

Reply:
It works without this code, so I think I can delete it. I checked and if I delete the `_bridgeExecutorOverrideToken`, the `reload` action works too.

Reply:
Let's remove the logic for `_bridgeExecutorOverrideTokenDevMenu`. We can keep the `_bridgeExecutorOverrideToken` until we fully remove the RCTWebSocketExecutor mode (will happen as part of the new arch)